shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedProcessorServiceImpl.java

```diff
@@ -16,4 +19,10 @@ public void createManagedProcessor(ProcessorDTO processorDTO) {
     public void deleteManagedProcessor(ProcessorDTO processorDTO) {
         // @TODO
     }
+
+    @Override
+    public List<ManagedProcessor> fetchAllManagedProcessors() {
+        // @TODO
+        return null;
```

Comment:
Unless this is to be completed before this PR is merged returning an empty collection may be preferential.

Reply:
@vaibhavjainwiz This implementation is required, so assuming there is more to come?

Reply:
done

Reply:
provided its concreat implementation.